app/transformers/api/v4/user_transformer.rb

```diff
@@ -4,7 +4,8 @@ def to_response(user)
       Api::V4::Transformer.to_response(user)
         .merge("registration_facility_id" => user.registration_facility.id,
                "phone_number" => user.phone_number,
-               "password_digest" => user.phone_number_authentication.password_digest)
+               "password_digest" => user.phone_number_authentication.password_digest,
+               "teleconsultation_phone_number" => user.teleconsultation_phone_number_with_isd)
```

Comment:
Are the plain database fields also exposed in the API automatically? Do we need to explicitly exclude them?

Reply:
The `teleconsultation_isd_code` needs to be removed, oops. Added in 88220b5c.